examples/federated-learning/tff/federated-learning-for-image-classification/container-image/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/jupyter/tensorflow-notebook:tensorflow-2.15.0
+
+WORKDIR "/home/jovyan"
```

Comment:
Can you add a comment explaining why we set this value?

Reply:
Maybe we don't need it at all as it is set in the base image

https://github.com/jupyter/docker-stacks/blob/main/images/docker-stacks-foundation/Dockerfile#L144

Reply:
If it's not needed, remove.

Reply:
It's needed as per Superlinter. Will add a comment on what it is